t/op/bop.t

```diff
@@ -407,6 +407,29 @@ SKIP: {
         do { use integer; 1 << ($bits - 1) } == -$cusp);
     ok (($cusp >> 1) == ($cusp / 2) &&
         do { use integer; abs($cusp >> 1) } == ($cusp / 2));
+
+    # GH #22412
+    for my $op (qw( & ^ | &. ^. |. )) {
+        my ($x, $y) = $op =~ /\./
+            ? ("z", ">")
+            : (0x7a, 0x3e);
```

Comment:
I believe this should be

`: (uni_to_native(0x7a), uni_to_native(0x3e));`

Reply:
I'm 99.5% sure that's wrong, but why do you think so?

Reply:
The code looks fine to me.

The tests don't pre-determine the result of the bitop, and in the numeric case which you've described changing it shouldn't differ between character encodings anyway as mauke points out.

If it is failing adding some Devel::Peek Dump()s for `$x`, `$y` and `$expected` to each case might give some idea what's going on.

Reply:
I had to do some workarounds to test this, so it is possible I screwed that up.  So merge it, and I'll test it later.